Formula/breakpad.rb

```diff
@@ -0,0 +1,69 @@
+class Breakpad < Formula
+  desc "Components implement a crash-reporting system."
+  homepage "https://chromium.googlesource.com/breakpad/breakpad/"
+
+  url "https://chromium.googlesource.com/breakpad/breakpad",
+      :using => :git,
+      :revision => "67d5567177301d0c24303f26ad119ab7bd7fab40"
+  version "0.1.0"
+
+  head "https://chromium.googlesource.com/breakpad/breakpad",
```

Comment:
IIRC you can actually use `https://chromium.googlesource.com/breakpad/breakpad.git` without that being a problem. It accepts both.